core/src/main/python/create.py

```diff
@@ -337,16 +391,29 @@ def main(model_context):
             archive_helper.extract_all_database_wallets()
 
         creator = DomainCreator(model_dictionary, model_context, aliases)
-        creator.create()
 
-        if has_atp:
-            rcu_properties_map = model_dictionary[model_constants.DOMAIN_INFO][model_constants.RCU_DB_INFO]
-            rcu_db_info = RcuDbInfo(model_context, aliases, rcu_properties_map)
-            atp_helper.fix_jps_config(rcu_db_info, model_context)
-        elif has_ssl:
-            rcu_properties_map = model_dictionary[model_constants.DOMAIN_INFO][model_constants.RCU_DB_INFO]
+        if dictionary_utils.get_dictionary_element(model_dictionary, model_constants.DOMAIN_INFO) is not None and \
+           dictionary_utils.get_dictionary_element(model_dictionary[model_constants.DOMAIN_INFO],
+                                                    model_constants.RCU_DB_INFO) is not None:
+            rcu_properties_map = dictionary_utils.get_dictionary_element(model_dictionary[model_constants.DOMAIN_INFO]
+                                                                         ,model_constants.RCU_DB_INFO)
             rcu_db_info = RcuDbInfo(model_context, aliases, rcu_properties_map)
-            ssl_helper.fix_jps_config(rcu_db_info, model_context)
+        else:
+            # create empty rcu_db_info for cli case
+            rcu_db_info = RcuDbInfo(model_context, aliases, None)
+
```

Comment:
There's a static method that does all these checks:
```
        rcu_db_info = rcudbinfo_helper.create(model_dictionary, model_context,aliases)
```

Reply:
Thanks, fixed